collect_stata/write_json.py

```diff
@@ -202,10 +202,44 @@ def generate_statistics(
 
     return metadata
 
+def update_metadata(
+    metadata: List[Dict[str, Union[str, Dict[str, List[Union[int, str, bool]]]]]],
+    metadata_de: List[Dict[str, Union[str, Dict[str, List[Union[int, str, bool]]]]]],
```

Comment:
We updated to python 3.8 on the develop branch. You can use TypedDicts to make this more readable and more precise. https://mypy.readthedocs.io/en/latest/more_types.html#typeddict